src/theme/NavbarItem/DefaultNavbarItem.js

```diff
@@ -0,0 +1,2 @@
+import OriginalDefaultNavbarItem from "@theme-original/NavbarItem/DefaultNavbarItem";
+export default OriginalDefaultNavbarItem;
```

Comment:
This was fun, I learned all about the concept of ["swizzling"](https://docusaurus.io/docs/swizzling), which is how we customize core docusaurus bits. 

In this file, and several below it, we're literally wrapping the original docusaurus implementation. This seems silly, but we _have_ to do it, because further down we want to override the implementation of another component _in this same folder_. If you replace one item inside of `NavbarItem`, you have to replace all of them. And since we don't actually want to change the implementation, we wrap them.

Reply:
(I also learned that [it's not just a made-up word](https://en.wikipedia.org/wiki/Swizzling_(computer_graphics)))